class BaseExceptionError(Exception):
    pass


class NotFoundError(BaseException):
    pass


class AlreadyExistError(BaseException):
    pass
